interp/interp.mllib

```diff
@@ -13,7 +13,6 @@ Impargs
 Implicit_quantifiers
 Constrintern
 Modintern
-Constrextern
 Coqlib
```

Comment:
What is the motivation for moving constrextern to printing? By consistency, constrintern should go in parsing then, and detyping should go in some new detyping directory?
It seems to me that it is easier to explain that pretyping is both about type inference and type discarding, and that interp is both about interpretation and uninterpretation that explaining that constrextern is about printing.

Unless there is a technical motivation also.